src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -371,6 +371,38 @@ prefetch('/about');
 
 <ReadMore>Learn more about [prefetching](/en/guides/prefetch/).</ReadMore>
 
+### Removed: `rewrite()` from Actions context
+
+<SourcePR number="14477" title="feat!: remove rewrite from action context"/>
+
+In Astro 5.5.6, the `ActionAPIContext.rewrite` method was deprecated because custom endpoints should be used where you might want to use rewrites.
```

Comment:
```suggestion
In Astro 5.5.6, the `ActionAPIContext.rewrite` method was deprecated because custom endpoints should be used instead of rewrites.
```
cleaner!